### PR TITLE
Add import pattern for Groovy.

### PR DESCRIPTION
**Pull request by PerilousBooklet**

### CODE REVIEW

**Review by Guldoman — COMMENTED** [review]
Needs a version bump.

File: manifest.json [hunk 1]
```diff
@@ -162,7 +162,7 @@
       "id": "meta_languages",
       "mod_version": "3",
       "type": "meta",
-      "version": "0.1.13"
+      "version": "0.1.15"
```
Comment (Guldoman) [inline manifest.json:167]: Version bump of the plugin, not `meta_languages`.

**Comment by PerilousBooklet** [comment]
The meta_languages version should be 0.14, right?

**Comment by Guldoman** [comment]
> The meta_languages version should be 0.14, right?

Yep, at the moment it is.

**Pull request merged**